src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,40 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate dateFromDate = LocalDate.of(Integer.parseInt(dateFrom.substring(6, 10)),
+                Integer.parseInt(dateFrom.substring(3, 5)),
+                Integer.parseInt(dateFrom.substring(0, 2)));
```

Comment:
Create formatter: 
private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy"); 
And use it like this:
LocalDate.parse(dateFrom, FORMATTER)
```suggestion
        LocalDate dateFromDate = LocalDate.parse(dateFrom, FORMATTER);
```